README.md

```diff
@@ -63,1498 +104,468 @@ Delegating a capability is like the statement `message`. Task is akin to `messag
-[dag-cbor]: https://ipld.io/specs/codecs/dag-cbor/spec/
-[ipld representation]: https://ipld.io/docs/schemas/features/representation-strategies/
-[lazy-vs-eager]: #112-lazy-vs-eager-evaluation
-[invoker]: #211-invoker
-[executor]: #212-executor
-[task]: #3-task
-[authorization]: #4-authorization
-[invocation]: #5-invocation
-[result]: #6-result
-[effect]: #7-effect
-[receipt]: #8-receipt
-[pipelines]: #9-pipelines
-[await]: #await
+Thanks to [Rod Vagg] for the clarifications on IPLD Schema implicits and the general IPLD worldview
+ 
+<!-- External Links -->
+
+[Agoric]: https://agoric.com/
+[BCP 14]: https://www.rfc-editor.org/info/bcp14
+[Bacalhau]: https://www.bacalhau.org/
+[Blaine Cook]: https://github.com/blaine
+[Brooklyn Zelenka]: https://github.com/expede/
+[Cap 'n Proto RPC]: https://capnproto.org/
+[CapTP]: http://erights.org/elib/distrib/captp/index.html
+[Christine Lemmer-Webber]: https://github.com/cwebber
+[DAG House]: https://dag.house
+[DAG-CBOR]: https://ipld.io/specs/codecs/dag-cbor/spec/
+[DID]: https://www.w3.org/TR/did-core/
+[Delegation]: https://github.com/ucan-wg/delegation
+[E-lang Mailing List, 2000 Oct 18]: http://wiki.erights.org/wiki/Capability-based_Active_Invocation_Certificates
+[Electronic Rights Transfer Protocol (ERTP)]: https://docs.agoric.com/guides/ertp/
+[Fission]: https://fission.codes/
+[Haskell]: https://en.wikipedia.org/wiki/Haskell
+[IEEE-754]: https://ieeexplore.ieee.org/document/8766229
+[IPLD]: https://ipld.io/
+[IPVM]: https://github.com/ipvm-wg
+[Irakli Gozalishvili]: https://github.com/Gozala
+[JS Number]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number
+[Luke Marsen]: https://github.com/lukemarsden
+[Marc-Antoine Parent]: https://github.com/maparent
+[Mark Miller]: https://github.com/erights
+[OAuth 1]: https://oauth.net/1/
+[OCapN]: https://github.com/ocapn/
+[Oxide Computer]: https://oxide.computer/ 
+[Philipp Krüger]: https://github.com/matheus23/
+[Protocol Labs]: https://protocol.ai
+[Quinn Wilton]: https://github.com/QuinnWilton
+[Robust Composition]: http://www.erights.org/talks/thesis/markm-thesis.pdf
+[Rod Vagg]: https://github.com/rvagg/
+[Simon Worthington]: https://github.com/simonwo
+[Spritely Institute]: https://spritely.institute/news/introducing-a-distributed-debugger-for-goblins-with-time-travel.html
+[UCAN Delegation]: https://github.com/ucan-wg/delegation/
+[UCAN Promise]: https://github.com/ucan-wg/promise/
+[URI]: https://en.wikipedia.org/wiki/Uniform_Resource_Identifier
+[Witchcraft Software]: https://github.com/expede
+[Zeeshan Lakhani]: https://github.com/zeeshanlakhani
+[`data`]: https://en.wikipedia.org/wiki/Data_URI_scheme
+[`ipfs`]: https://docs.ipfs.tech/how-to/address-ipfs-on-web/#native-urls
+[`magnet`]: https://en.wikipedia.org/wiki/Magnet_URI_scheme
+[capability systems]: https://en.wikipedia.org/wiki/Capability-based_security
+[designation without authorization]: https://srl.cs.jhu.edu/pubs/SRL2003-02.pdf
```

Comment:
```suggestion
[designation with authorization]: https://srl.cs.jhu.edu/pubs/SRL2003-02.pdf
```
This is hard to grok, since in that article there is a principle of "_NO_ designation without authority", inverted when applied as [an e2e design principle] to become "designation with authority" on page 3.